src/2geom/interval.h

```diff
@@ -113,6 +113,15 @@ class Interval
         if (v >= max()) return 1;
         return timeAt(v);
     }
+    /// Find furthest time in [0,1] that maps to the given value. */
+    Coord furthestTime(Coord v) {
```

Comment:
The furthest time will always be 0 or 1, depending on which endpoint is closer, so this could be a lot simpler.